data/help/texts.json

```
@@ -170,16 +171,16 @@
     "order": 13,
     "name": "<n|N>: Combat",
     "messages": [
-      "Zombies in cities will spawn at the start of the game but may also wander around.  All monsters are represented by letters on your screen; a list of monster names, and their positions relative to you, is displayed on the right side of the screen.  The so called compass will display monsters that you see but are outside the screen.",
-      "To attack a monster with a melee weapon, simply move into them.  The time it takes to attack depends on the size and weight of your weapon.  Small, light weapons are the fastest; unarmed attacks increase in speed with your Unarmed Combat skill, and will eventually be VERY fast.  A successful hit with a bashing weapon may stun the monster temporarily.  A miss may make you stumble and lose movement points.  If a monster hits you, your clothing may absorb some damage, but you will absorb the excess.",
+      "Zombies in cities will spawn at the start of the game, but may also wander around.  All monsters are represented by letters (or small images called tiles) on your screen; a list of monster names, as well as their positions relative to you, is displayed on the right side of the screen.  The 'compass' will display monsters that you see but are currently off the screen.",
```

```suggestion
      "Zombies in cities will spawn at the start of the game, but may also wander around.  All monsters are represented by letters (or small images in the Tiles version) on your screen; a list of monster names, as well as their positions relative to you, is displayed on the right side of the screen.  The 'compass' will display monsters that you see but are currently off the screen.",
```
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/TILESET.md#tile

---

Oops. Thanks for catching that!